_posts/developers/apps/2021-05-25_libraries.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: for-developers
+title: Tools and libraries overview
+permalink: developers/tools
```

Comment:
So this line is what determines the final URL - and it's currently the same as 2019-01-01-00_overview.md, which is why I can't see it yet at https://deploy-preview-579--solidproject-dot-org.netlify.app/developers/tools

I think if you just delete the two files in  `_posts/developers/apps/tools/`, this should be good to go.

Reply:
Hmm, deleted the two old ones.  Now preview gives  a 404.  Maybe _overview.md is a magic name ... I'll try renaming.

Reply:
@Vinnl - sorry, unable to get anything other than a 404 in the preview.  I added the new file, deleted the two old files, then renamed the new file to overview.md ... still 404 in the preview

Reply:
Hmm... I'm kinda stumped too - as far as I can see this should do it. @justinwb do you see anything in the build log that might indicate why there's nothing at the above URL?

Reply:
closing this and starting with a clean PR

Reply:
Started fresh, same deal, moving the discussion to https://github.com/solid/solidproject.org/pull/580